ansible-scylla-node/tasks/common.yml

```diff
@@ -67,7 +67,7 @@
 
     - set_fact:
         io_conf: "{{ _io_conf.stdout_lines[0] }}"
-  when: scylla_io_probe|bool == True and (scylla_io_probe_dc_aware|bool == False or dc_to_node_list[dc][0] == inventory_hostname)
+  when: io_properties is not defined and scylla_io_probe|bool == True and (scylla_io_probe_dc_aware|bool == False or dc_to_node_list[dc][0] == inventory_hostname)
```

Comment:
Your usage of `io_settings` term in the patch description is confusing.
As far as I know there is no such term.

You probably wanted to write "I/O settings" instead.